Makefile

```diff
@@ -77,7 +77,7 @@ test-in-docker:
 	# Privileged needed for docker-in-docker so integ tests pass
 	docker run -v "$(shell pwd):/go/src/github.com/aws/amazon-ecs-agent" --privileged "amazon/amazon-ecs-agent-test:make"
 
-run-functional-tests: test-registry
+run-functional-tests: docker test-registry
```

Comment:
A make target for this would be great. Running `source ./scripts/shared_env` and `go test -tags functional -v ./... -run RunManyTasks` subsequently did not work for me.